test/functional/test-url-replacements.js

```diff
@@ -1312,4 +1318,18 @@ describes.sandboxed('UrlReplacements', {}, () => {
           'NDrFP4BtPQJMyxE4jb9FDlp37OJL');
     });
   });
+
+  describe('formatted timestamp', () => {
+    it('should return correct formatted timestamp', () => {
+      const fakeTime = 1499979336612;
+      const offset_ = new Date(fakeTime).getTimezoneOffset() * 60000;
+      sandbox.useFakeTimers(fakeTime + offset_);
+      return expandAsync(
+          '?tsf=TIMESTAMP_FORMATTED').then(res => {
+            expect(encodeURIComponent('2017-07-13 20:55:36'))
+                .to.equal('2017-07-13%2020%3A55%3A36');
+            expect(res).to.equal('?tsf=2017-07-13%2020%3A55%3A36');
```

Comment:
`return expect(expandAsync('?tsf=TIMESTAMP_FORMATTED')).to.eventually.equal('?tsf=' + encodeURIComponent('2017-07-13 20:55:36'))`